azure-functions/acbs-function/api.js

```diff
@@ -24,23 +25,50 @@ const getACBS = async (apiRef) => {
       headers: {
         'Content-Type': 'application/json',
       },
-    }).catch((err) => ({
-      status: !!err.response
-        ? err.response.status
-        : err,
-    }));
-    return response;
+    }).catch((e) => {
+      console.error('Error calling GET to ACBS');
+      return {
+        status: e.response ? e.response.status : e,
+        data: { error: e.response ? e.response.data : e },
+      };
+    });
   }
   return {};
 };
 
-const putToACBS = async (apiRef, acbsInput, etag) => {
+const postToAcbs = async (apiRef, acbsInput) => {
   if (!!apiRef && !!acbsInput) {
-    const additionalHeader = etag ? {
-      'If-Match': etag,
-    } : null;
+    return axios({
+      method: 'post',
+      url: `${process.env.MULESOFT_API_UKEF_TF_EA_URL}/${apiRef}`,
+      auth: {
+        username: process.env.MULESOFT_API_KEY,
```

Comment:
Can we assign the env variables to a const  variable at the top of the file? I see these values being referenced a few times in this file